components/inputs/test/input-search.test.js

```diff
@@ -98,27 +99,64 @@ describe('d2l-input-search', () => {
 			expect(detail.value).to.equal('');
 		});
 
-		it('should fire "search" event when search input changes in search-on-input mode', async() => {
-			const elem = await fixture(searchOnInputFixture);
-			elem.value = 'foobar';
-			setTimeout(() => getTextInput(elem).dispatchEvent(new Event('input')));
-			const { detail } = await oneEvent(elem, 'd2l-input-search-searched');
-			expect(detail.value).to.equal('foobar');
-		});
+		describe('fake timer', () => {
 
-		it('should fire "search" event only once after two consecutive input events', async() => {
```

Comment:
I moved these two tests inside the suite where fake timers are used. That way we can advance the clock programatically instead of waiting `50ms` in the test and slowing everything down.